Sprint-2/debug/1.js

```diff
@@ -1,8 +1,10 @@
 // Predict and explain first...
 
 function sum(a, b) {
-  return;
-  a + b;
+  return a + b;
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+
+// teh result will be the text and "undefined" 
+// because we returned the function without giving it functionality, (a+b) should be infront of return
```

Comment:
Yes, the ; after the return on line 4, finishes the statement so the function just returns `undefined`

Reply:
I checked it and it returns correct result